resources/templates/md/pages/about.md

```diff
@@ -3,13 +3,20 @@
  :page-index 1
  :navbar? true}
 
-The Bangalore Clojure Group hosts one of the most active (since 2013) meetups for Clojure in India, each of our meetups are conducted in different locations with different set of speakers every month.Our meetups are scheduled conveniently on every first Saturday of the month.
+The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
```

Comment:
Meetups started much earlier than 2013 - check https://groups.google.com/forum/#!forum/bangalore-clj older posts for details.

Reply:
fixed